runtime/src/lib.rs

```diff
@@ -518,6 +535,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
+	pallet_contracts::Migration<Runtime>,
```

Comment:
Why this migration is needed?

Reply:
I found it in substrate-contracts-node. As I understand it used to migrate between different storage versions. Maybe it's useless in our case.